tests/schemas/test_emergency_contact_schema.py

```diff
@@ -12,3 +15,49 @@ def test_valid_payload(self, empty_test_db):
         no_validation_errors = {}
 
         assert no_validation_errors == EmergencyContactSchema().validate(valid_payload)
+
+    def test_validate_empty_contact_number(self, empty_test_db):
+        invalid_payload = {
+            'name': 'emergency contact name',
+            'contact_numbers': [{}]
+        }
+        validation_errors = EmergencyContactSchema().validate(invalid_payload)
+
+        assert 'contact_numbers' in validation_errors
+
+    def test_validate_multiple_contact_numbers(self, empty_test_db):
+        invalid_payload = {
```

Comment:
Perhaps this should be called `valid_payload`

Reply:
I agree, I really need to be more careful haha, sorry!

Reply:
No worries, it happens to me all the time.